### ISSUE TITLE
Dev board index doesn't point at latest unstable version

### ISSUE THREAD

**Issue by tyeth** [espressif/arduino-esp32#9489]
### Version

3.0.0-alpha3

### IDE Name

arduino-cli

### Operating System

win11

### Flash frequency

80

### PSRAM enabled

yes

### Upload speed

115200

### Description

Can't install 3.0.0-alpha3

### Sketch

```cpp
any
```

### Debug Message

```plain
arduino-cli.exe --additional-urls=https://espressif.github.io/arduino-esp32/package_esp32_dev_index.json core install esp32:esp32@3.0.0-alpha3

Error during install: Platform 'esp32:esp32@3.0.0-alpha3' not found: required version 3.0.0-alpha3 not found for platform esp32:esp32
```

```
arduino-cli.exe --additional-urls=https://espressif.github.io/arduino-esp32/package_esp32_dev_index.json core install esp32:esp32
Platform esp32:esp32@2.0.15 already installed
```
```


### Other Steps to Reproduce

_No response_

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by brentru** [comment]
@me-no-dev As you released "[Arduino Release v2.0.15 based on ESP-IDF v4.4.7](https://github.com/espressif/arduino-esp32/releases/tag/2.0.15)", could you take a look at this issue? Thank you in advance.

**Comment by lucasssvaz** [comment]
@me-no-dev I think the bot replaced it in https://github.com/espressif/arduino-esp32/commit/544b4a2b209293a26a14ee0b096ae8452db34846

**Comment by me-no-dev** [comment]
yup... this is the first time we release lower version after alpha with higher one... so script failed :) we need to rework it in the future. We are about to release RC1 of 3.0.0 so it will be back to normal in no time

**Comment by VojtechBartoska** [comment]
same issue as https://github.com/espressif/arduino-esp32/issues/9492. closing as fixed by 3.0.0-RC1 release.

**Comment by tyeth** [comment]
Thanks all, appreciate the quick release
